client/style.styl

```diff
@@ -642,4 +642,21 @@ body.is-seed .show-leech
   margin-right: 42px
   max-width: 50%
 
+.telemetry-logs > div
+  margin-bottom: 16px
+
+.telemetry-logs a
+  display: inline-block
+  padding: 2px 6px
+  margin: 2px
+  border-radius: 2px
+  background-color: #ddd
+
+.telemetry-charts h3
+  margin-top: 20px
+  margin-bottom: 0
+
+.telemetry-charts svg
+  height: 300px !important
```

Comment:
These can go into a `<style>` tag that's just on the telemetry page, unless we want to make these charts public.